pac4j-core/src/main/java/org/pac4j/core/util/CommonHelper.java

```diff
@@ -236,102 +236,156 @@ public static String toString(final Class<?> clazz, final Object... args) {
     }
 
     /**
+     * Extract the prefix of the name.
+     *
+     * @param name the name
+     * @return the prefix
+     */
+    protected static String extractPrefix(final String name) {
+        int prefixEnd = name.indexOf(":");
+        String prefix = null;
+        if (prefixEnd != -1) {
+            prefix = name.substring(0, prefixEnd);
+        }
+        return prefix;
+    }
+
+    /**
+     * Add a slash at the beginning of a path if missing.
+     *
+     * @param path the path
+     * @return the completed path
+     */
+    protected static String startWithSlash(final String path) {
+        if (!path.startsWith("/")) {
+            return "/" + path;
+        } else {
```

Comment:
You would remove the `else` clause to directly have `return path;`, wouldn't you?

I think it's clearer this way, but it's just a personal feeling.